src/renderer/store/modules/settings.js

```diff
@@ -172,6 +172,7 @@ const state = {
   baseTheme: 'system',
   mainColor: 'Red',
   secColor: 'Blue',
+  defaultAutoplayInterruptionInterval: 3,
```

Comment:
Other interval settings are seconds by default
Let's name it so it's clear that the value is hours not seconds
```suggestion
  defaultAutoplayInterruptionIntervalHours: 3,
```

Reply:
Done!